hive/src/main/java/org/apache/iceberg/hive/HiveCatalog.java

```diff
@@ -329,7 +350,9 @@ private boolean alterHiveDataBase(Namespace namespace,  Database database) {
 
     try {
       Database database = clients.run(client -> client.getDatabase(namespace.level(0)));
-      return convertToMetadata(database);
+      Map<String, String> metadata = convertToMetadata(database);
+      LOG.debug("Loaded metadata for namespace {}", namespace);
```

Comment:
Sorry, but I didn't catch this before. This is logging a property map that could contain sensitive items. Could you log just the keys instead?

Reply:
Thanks for the quick turn-around!

Reply:
I did not log the metadata loaded, as I should.
Added the log of the loaded metadata keys, like this:
```
Loaded metadata for namespace dbname_load found [owner, location, comment, group]
```

Reply:
And feel free to ask changes if they are needed. Better now, than 1 year from now when some info is leaked!